docs/developers/manual.md

```diff
@@ -2,58 +2,26 @@
 
 ## Getting started
 
-### Mac OS
+## Build System
 
 Install Bazel:
-```shell
-$ brew install bazel
-```
-
-### Create the dev database
-
-Add the following line to the `.env` file at the root of the
-codebase.
-
-```
-DATABASE_URL="sqlite3:/$HOME/.sematic/db.sqlite3"
-```
+- [MacOS](https://bazel.build/install/os-x#install-on-mac-os-x-homebrew)
+- [Ubuntu](https://bazel.build/install/ubuntu#install-on-ubuntu)
+- [RHEL](https://bazel.build/install/redhat)
 
-Then create the database:
+## Dev Tools
 
 ```shell
-$ mkdir ~/.sematic
-$ touch ~/.sematic/db.sqlite3
+$ make install-dev-deps
 ```
 
-And run all migrations
-```shell
-$ bazel run //sematic/db:migrate -- up --verbose
-```
-
-## Add third-party pip dependency
+## Add a third-party pip dependency
 
-Add dependency to `requirements/requirements.in`, and if possible, set a fixed version.
+Add your dependency to `requirements/requirements.in`, and if possible, set a fixed version.
```

Comment:
Should we NOT advise to set a fixed version unless necessary?

Reply:
Done.